website/documentation/guides/install_gardener/setup-seed/_index.md

```diff
@@ -8,17 +8,33 @@ scope: operator
 aliases: ["/readmore/seed", "/050-tutorials/content/howto/setup-seed"]
 ---
 
-# The Seed Cluster
+# Deprecated
+Please note that in the meanwhile there is an easier way to register a Shoot cluster as a Seed. 
+These clusters are called `Shooted Seeds` in the Gardener terminology. Please [see this document](https://github.com/gardener/gardener/blob/master/docs/usage/shooted_seed.md) for more information.
 
-The [landscape-setup-template](https://github.com/gardener/landscape-setup-template) is meant to provide an as-simple-as-possible Gardener installation. Therefore it just registers the cluster where the Gardener is deployed on as a seed cluster. While this is easy, it might be insecure. Clusters created with Kubify don't have network policies, for example. See [Hardening the Gardener Community Setup](/readmore/hardening) for more information. 
+To register Seed cluster in "fenced" environments (e.g. behind a firewall), please refer to [this document](https://gardener.cloud/documentation/guides/install_gardener/deploy-gardenlet/).
 
-To have network policies on the seed cluster and avoid having the seed on the same cluster as the Gardener, the easiest option is probably to simply create a shoot and then register that shoot as seed. This way you can also leverage other advantages of shooted clusters for your seed, e.g. autoscaling.
+
+# Seed Cluster created by the landscape setup
```

Comment:
This heading seems reads as title on github. On the other hand the title in front-matter is "Register a Seed Cluster" and this is confusing.
If I get it right this section is motivation for laying out the procedure that follows. 
Either use the Title as h1 heading (which will make it redundant with the current website documentation pages layout and doesn't look nice, but that's something we have to figure in general), or use something like Motivation or Introduction for header or just skip the header at all (either of which may not look nice if you read it on github because the title will be missing)